packages/table/src/TableWithVS.story.tsx

```diff
@@ -1,28 +1,24 @@
 import React from 'react';
 import { VirtualItem } from 'react-virtual';
```

Comment:
I feel like we should export this too? I don't want anyone having to use any external package..